dune/xt/la/container/common.hh

```diff
@@ -762,49 +837,63 @@ public:
   } // CommonSparseMatrix(...)
 
   template <int ROWS, int COLS>
-  explicit operator Dune::FieldMatrix<ScalarType, ROWS, COLS>() const
+  explicit operator std::unique_ptr<Dune::FieldMatrix<ScalarType, ROWS, COLS>>() const
```

Comment:
Could you comment on why you return a `std::unique_ptr` here instead of the value (thus having a different signature than the `operator` below)? I woul think that with RTO we could just return by value and create the unique_ptr on the outside, if required...